indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorSpec.java

```diff
@@ -132,6 +133,13 @@ public Map<String, Object> getContext()
     return context;
   }
 
+  @JsonIgnore
```

Comment:
Why JsonIgnore?  Jackson should only be using tagged fields anyway, so just not tagging it should be sufficient.